docs/reference/auto-updates.md

```diff
@@ -0,0 +1,14 @@
+---
+id: auto-updates
+title: "Auto-updates"
+description: "Some updates are applied to SaaS clusters automatically."
+---
+
+Camunda 8 SaaS customers can enable auto-updates. When enabled, the cluster will be updated once a new patch release is available.
+
+For Camunda 8 Saas, auto-updates are only for patch releases (x.y.**z**, where **z** is a patch release).
+
+Minor updates are not eligible for auto-updates due to the following:
```

Comment:
I would not add the reasons right now. I feel this plays not that well
Let's document the positive and instead only take the sentence that Max proposed.